demo_nodes_cpp/src/topics/listener_serialized_message.cpp

```diff
@@ -70,7 +70,7 @@ class SerializedMessageListener : public rclcpp::Node
   }
 
 private:
-  rclcpp::Subscription<rclcpp::SerializedMessage>::SharedPtr sub_;
+  rclcpp::Subscription<std_msgs::msg::String>::SharedPtr sub_;
```

Comment:
I think it's worth to add a note in the release notes about this API change: https://github.com/ros2/ros2_documentation/blob/rolling/source/Releases/Release-Humble-Hawksbill.rst

Reply:
I agree, at the time we were waiting on that page to be created. @audrow does it make sense to add it there now?